scripts/json.cmake

```diff
@@ -5,7 +5,7 @@ set(JSON_MultipleHeaders OFF)
 FetchContent_Declare(
     json
     GIT_REPOSITORY https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent.git
-    GIT_TAG        67e6070f9d9a44b4dec79ebe6b591f39d2285593
+    GIT_TAG        master
```

Comment:
again this is messing with the version control and can cause issues later down the road